backend/support/management/commands/run_2022.py

```diff
@@ -26,7 +26,8 @@ def handle(self, *args, **kwargs):
             sac = self.make_sac(gen)
             sac.submission_status = sac.STATUS.SUBMITTED
             sac.save()
-            sac.disseminate()
+            if not sac.cognizant_agency and not sac.oversight_agency:
```

Comment:
This change is correct. We don't want to run the entire `sac.disseminate()` process, we just want to run `sac.assign_cog_over()` if appropriate.